packages/aura-language-server/.vscode/settings.json

```diff
@@ -1,30 +1,20 @@
 {
-    "tslint.exclude": [
-        "**/src/html-language-service/htmlLanguageService.ts",
-        "**/src/html-language-service/htmlLanguageTypes.ts",
-        "**/src/html-language-service/parser/htmlParser.ts",
-        "**/src/html-language-service/parser/htmlScanner.ts",
-        "**/src/html-language-service/parser/htmlTags.ts",
-        "**/src/html-language-service/services/htmlCompletion.ts",
-        "**/src/html-language-service/utils/strings.ts",
-        "**/src/html-language-service/utils/paths.ts",
-        "**/src/html-language-service/utils/arrays.ts"
-    ],
-    "search.exclude": {
-        "**/node_modules": true,
-        "lib": true,
-        "resources": true
-    },
-    "terminal.integrated.env.osx": {
```

Comment:
These `terminal.integrated` settings used to be added by the Salesforce extensions but got deprecated last year.